### ISSUE TITLE
Network access not working on NetBSD/sparc64

### ISSUE THREAD

**Issue by connorwit** [skyjake/lagrange#207]
I was able to compile lagrange on NetBSD/sparc64. It renders the integrated pages well and fast with software rendering.

Everything seems to work, but it can't access and fetch pages from the internet. The loading bar stops at about 3/4 of its length and the loading indicator in the address bar will continue to rotate forever. No error will appear at some point.

Is there any advice on how to create some debug logs to get to the cause?

**Comment by skyjake** [comment]
This sounds like a socket or address lookup issue of some sort. Try compiling with the CMake option `TFDN_ENABLE_DEBUG_OUTPUT` set to `YES`, and run lagrange with `--echo`. It should print messages to stdout about how the connection is progressing.

**Comment by skyjake** [comment]
I was intrigued so I took a look in NetBSD 9.1 (amd64). Looks like the fixes for FreeBSD apply here too, so using libpthread instead of C11 threads seems to help a lot. Without the fix, no network requests work because they run in a background thread and the C11 threads behave somehow differently than what the_Foundation expects.
![netbsd_lagrange](https://user-images.githubusercontent.com/1225742/111446545-31882100-8715-11eb-9260-ad446f70c216.png)
There is something odd going on with command line options, though: the app is not getting any of them even though options are specified on the command line. This interferes with finding `resources.lgr` when running from the build directory.

But otherwise it seems up and running now. I've committed the changes to the dev branch (on git.skyjake.fi).
